client/src/dev-components/container-card/container-card.js

```diff
@@ -79,8 +78,6 @@ class ContainerCard extends React.Component {
   render() {
     const { instance } = this.props
 
-    const typeText = instance.runner === 'site' ? 'External site' : 'Instance'
```

Comment:
Should be re-added, I think I accidentally removed this in rebase

Reply:
atleast variable 'typeText' isn't used in anywhere in container-card.js, you can look into it more

Reply:
The usage was also accidentally removed, that's why it's not used

Reply:
reverted